cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/dataset/SearchRequest.java

```diff
@@ -175,7 +175,6 @@ public int getNumCursors() {
   public String getCursor() {
     return cursor;
   }
-
```

Comment:
nit: unintentional?

Reply:
Yes, should be addressed in [#11489](https://github.com/cdapio/cdap/pull/11489)